com.playeveryware.eos/Runtime/Core/Config/PlatformConfig.cs

```diff
@@ -266,6 +266,14 @@ protected PlatformConfig(PlatformManager.Platform platform) :
             Platform = platform;
         }
 
+        protected override void OnReadCompleted()
+        {
+            base.OnReadCompleted();
+
+            // If thread affinity is null then instantiate it.
+            threadAffinity = new();
```

Comment:
This should nullcheck / default check. As-is it doesn't do what this comment says it does.

Reply:
🤦 yeah, you'd think I'd have caught that 😆 

I've corrected it now.